src/components/layout/header/DesktopNavMenu.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+
+import { SecondaryNav } from '@/components/layout/header/SecondaryNav';
+import { Button } from '@/components/ui/Button';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from '@/components/ui/DropdownMenu';
+import { useMediaQuery } from '@/lib/hooks/useMediaQuery';
+import { EllipsisIcon } from 'lucide-react';
+import { useEffect, useState } from 'react';
+
+type DesktopNavMenuProps = {
+  className?: string;
+  t: {
+    open: string;
+    close: string;
+    storage: string;
+    shiftSchedule: string;
+  };
+};
+
+function DesktopNavMenu({ t }: DesktopNavMenuProps) {
+  const [open, setOpen] = useState(false);
+  const visible = useMediaQuery('(min-width: 48rem)');
+
+  useEffect(() => {
+    if (!visible) {
+      setOpen(false);
+    }
+  }, [visible]);
+
+  return (
+    <DropdownMenu open={open} onOpenChange={setOpen}>
+      <DropdownMenuTrigger asChild>
+        <Button
+          className='h-fit'
+          variant='nav'
+          size='none'
+          aria-label={open ? t.close : t.open}
+        >
+          <EllipsisIcon aria-hidden />
+        </Button>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent className='hidden min-w-[6rem] md:flex' align='end'>
+        <SecondaryNav
+          asDropDown
+          onClick={() => setOpen(false)}
+          t={{
+            storage: t.storage,
```

Comment:
have to click on the title in order to open page, clicking elsewhere does nothing.